apps/client/src/features/rundown/event-block/EventBlock.tsx

```diff
@@ -264,6 +267,11 @@ export default function EventBlock(props: EventBlockProps) {
     setSelectedEvents({ id: eventId, index, selectMode: editMode });
   };
 
+  const isNextDay =
+    previousStart === undefined || previousEnd === undefined
+      ? false
+      : checkIsNextDay(previousStart ?? 0, timeStart, calculateDuration(previousStart, previousEnd));
+
```

Comment:
question: we calculated this when we calculated the gap, maybe we could return an object of metadata like
```ts

export function calculateAccumulatedGap(): { gap: number, isNextDay: boolean }